readme.txt

```diff
@@ -37,6 +37,9 @@ https://demo.bmlt.app/crouton
 
 == Changelog ==
 
+= 3.26.2 =
+* Add meeting times to group table.
+
```

Comment:
if this is a bug fix for google maps we should include that in changelog/release notes, also needs to be 4.0.1